staging/src/k8s.io/pod-security-admission/test/testdata/baseline/v1.22/pass/sysctls1.yaml

```diff
@@ -19,3 +19,5 @@ spec:
       value: "0"
     - name: net.ipv4.ping_group_range
       value: 1 0
+    - name: net.ipv4.ip_unprivileged_port_start
+      value: "1024"
```

Comment:
you'll need to regenerate the testdata for all versions... 

run `UPDATE_POD_SECURITY_FIXTURE_DATA=true go test k8s.io/pod-security-admission/test` to update test fixtures